aiidalab_launch/instance.py

```diff
@@ -286,34 +286,45 @@ async def echo_logs(self) -> None:
     @staticmethod
     async def _init_scripts_finished(container: Container) -> None:
         loop = asyncio.get_event_loop()
-        LOGGER.debug("Waiting for init services to finish...")
+        LOGGER.info("Waiting for init services to finish...")
         result = await loop.run_in_executor(
             None, container.exec_run, "wait-for-services"
         )
         if result.exit_code != 0:
             raise FailedToWaitForServices(
                 "Failed to check for init processes to complete."
             )
-        LOGGER.debug("Init services finished.")
+        LOGGER.info("Init services finished.")
 
     @staticmethod
     async def _notebook_service_online(container: Container) -> None:
         loop = asyncio.get_event_loop()
-        LOGGER.debug("Waiting for notebook service to become reachable...")
+        LOGGER.info("Waiting for notebook service to become reachable...")
+        proto = "http"
```

Comment:
Can we give this variable a slightly more descriptive name? Maybe something like:
```suggestion
        assumed_protocol = "http"
```